remc2/engine/GameRenderInterface.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#ifndef GAME_RENDER_INTERFACE
+#define GAME_RENDER_INTERFACE
+
+#include <cstdint>
+#include "engine_support.h"
+#include "ViewPort.h"
+//#include "Type_E9C38_Smalltit.h"
+//#include "Type_F2C20ar.h"
+
+class GameRenderInterface 
+{
+public:
+	// Empty virtual destructor for proper cleanup
+	//virtual GameRenderInterface(uint8_t renderThreads, bool assignToSpecificCores, int rendererType);
+	//GameRenderInterface() {}
+	virtual ~GameRenderInterface() {}
+	//virtual void Init(uint8_t renderThreads, bool assignToSpecificCores) = 0;
```

Comment:
We have a complete history, I think (where we can) we should delete commented out lines